main/src/cgeo/geocaching/ui/dialog/Dialogs.java

```diff
@@ -1077,6 +1078,10 @@ public void onSlide(final @NonNull View bottomSheet, final float slideOffset) {
         return new MaterialAlertDialogBuilder(newContextThemeWrapper(context));
     }
 
+    public static AlertDialog.Builder newBuilder(final Context context, final @StyleRes int resId) {
```

Comment:
I wonder if it makes sense to restrict the possibilities for possible dialog styles (and thus provide e.g. some enum as parameter instead of the "raw" resId)? My fear is that this encourages developers in the future to invent more and more "dialog styles"

Reply:
Let's keep this in mind and see at the end of our current UI overhaul how many styles we need, and if we can stay (or come back to) a single `AlertDialog` style applied for all of our dialogs, or if we need this parameter for every dialog, with a specific value. (I'm thinking of all those user-defined dialogs.)